### PR TITLE
Add graphing

### PR DESCRIPTION
**Pull request by neuromantic99**

### CODE REVIEW

File: components/playgroundChatbot.tsx [hunk 1]
```diff
@@ -354,10 +354,7 @@ export default function PlaygroundChatbot(props: {
                 functionJsonResponse &&
                 typeof functionJsonResponse === "object"
               ) {
-                contentString = convertToRenderable(
-                  functionJsonResponse,
-                  `${functionNameToDisplay(chatItem?.name ?? "")} result`
-                );
+                contentString = chatItem.content;
```
Comment (Henry-Pulver) [inline components/playgroundChatbot.tsx:357]: You missed one champ

File: public/presets/2/demo-openapi-spec.json [hunk 2]
```diff
@@ -252,6 +252,56 @@
...
+                      }
+                    }
+                  }
+                }
+              }
+            }
+          }
+        }
+      }
+    },
```
Comment (Henry-Pulver) [inline public/presets/2/demo-openapi-spec.json:304]: Was this deliberate?

File: components/playgroundChatbot.tsx (hunk 1 above)
Comment (neuromantic99) [reply components/playgroundChatbot.tsx:357]: don't understand this comment 😆

File: public/presets/2/demo-openapi-spec.json (hunk 2 above)
Comment (neuromantic99) [reply public/presets/2/demo-openapi-spec.json:304]: yeah that's the simple graphable endpoint

File: components/playgroundChatbot.tsx (hunk 1 above)
Comment (Henry-Pulver) [reply components/playgroundChatbot.tsx:357]: This change was in the last PR I reviewed :smile:

Comment (Henry-Pulver) [reply components/playgroundChatbot.tsx:357]: Ohhh you're right :facepalm: 

Muppet of the day award goes to me :joy:

**Pull request merged**